basic/06c_kinematics_inv_kinematics.cpp

```diff
@@ -106,8 +107,18 @@ int main()
 
   std::cout << "Target position: " << std::endl << target_xyz.transpose() << std::endl;
   std::cout << "IK joint angles: " << std::endl << ik_result_joint_angles.transpose() << std::endl;
-  model->getEndEffector(ik_result_joint_angles, transform);
-  std::cout << "FK of IK joint angles: " << std::endl << transform.topRightCorner<3,1>().transpose() << std::endl << std::endl;
+  hebi::robot_model::Matrix4dVector transforms;
+  model->getFK(HebiFrameTypeOutput, ik_result_joint_angles, transforms);
+
+  transforms.emplace(transforms.begin(),Eigen::Matrix<double,4,4>::Identity());
+  std::vector<std::vector<double>> lines_x;
+  std::vector<std::vector<double>> lines_y;
+  std::vector<std::vector<double>> lines_z;
+
+  for(size_t j = 0; j < transforms.size(); ++j) {
+    plot_3dtriad(transforms[j],&lines_x,&lines_y,&lines_z, static_cast<bool>(j));
+  }
+  plt::pause(1);
```

Comment:
This is surprisingly concise -- I still think we might want to add a "plot utility functions" class, and have this wrapped up as a "plotFrames()" call.  We could even do the same with the earlier plotting from logs functions, before this is added to the C++ API...